### PR TITLE
40 add input to filename + benchrunner function

### LINKED ISSUE #40: Add input to filename
Input = Benchmark iteration/workload

### PR DESCRIPTION
**Pull request by sabj17**
Adds input to the title of the results files:
Implements a counting function to count length of input (for e.g. mergesort)
Implements runbenchmark function to eliminate bloat shell code @Jakobzs

### CODE REVIEW

**Comment by sabj17** [comment]
Test on SUT passed...

File: benchRunners/empty.sh [hunk 1]
```diff
@@ -1,64 +1,45 @@
+source ./benchRunners/bench_func.sh
+
 testName="empty"
 folder="empty"
 count=1000
 
+#runbenchmark language testName cmd (input) (inputSize)
```
Comment (Villiamfj) [inline benchRunners/empty.sh]: karl koder foreslår yeet

File: benchRunners/mergeSort.sh [hunk 2]
```diff
@@ -1,65 +1,48 @@
+source ./benchRunners/count_input_params.sh
+source ./benchRunners/bench_func.sh
+
 testName="mergeSort"
 folder="mergesort"
 count=1000
 mergeInput=`cat benchRunners/mergeSortParam` # getting input from file
+inputLength=$(count_params $mergeInput)
+
+#runbenchmark language testName cmd (input) (inputSize)
```
Comment (Villiamfj) [inline benchRunners/mergeSort.sh]: karl koder foreslår yeet igen

**Review by Villiamfj — CHANGES_REQUESTED** [review]

**Review by Villiamfj — APPROVED** [review]

**Review by Villiamfj — APPROVED** [review]
Det er kørt på seff 👍

**Review by Jakobzs — APPROVED** [review]

**Pull request merged**